subgraph.yaml

```diff
@@ -60,3 +60,25 @@ dataSources:
         - event: Unpause(indexed address)
           handler: handleUnpause
       file: ./src/agent-factory.ts
+  - kind: ethereum
+    name: AgentMech
+    network: gnosis
+    source:
+      address: "0x77af31de935740567cf4ff1986d04b2c964a786a"
+      abi: AgentMech
+      startBlock: 30776800
```

Comment:
I Only added this mech as source, since this the one being used currently? Should add all of them?

Reply:
Yes please add all of the ones we are currently using. Check with Ardian which ones they are

Reply:
Addressed [236d6cf](https://github.com/valory-xyz/ai-registry-mech-graph/pull/2/commits/236d6cf6f26c1262ea233bb56986acd37732ee32)